xtractmime/mimegroups.py

```diff
@@ -0,0 +1,78 @@
+from _patterns import ARCHIVE_TYPES, FONT_TYPES, JAVASCRIPT_TYPES
+
+
+def is_image_mime_type(mime_type: bytes) -> bool:
+    if mime_type.startswith(b"image/"):
+        return True
+
+    return False
+
+
+def is_audio_video_mime_type(mime_type: bytes) -> bool:
+    video_types = (b"audio/", b"video/")
+    if mime_type.startswith(video_types) or mime_type == b"application/ogg":
+        return True
+
+    return False
+
+
+def is_font_mime_type(mime_type: bytes) -> bool:
+    if mime_type.startswith(b"font/") or mime_type in FONT_TYPES:
+        return True
+
+    return False
+
+
+def is_zip_based_mime_type(mime_type: bytes) -> bool:
+    if mime_type.endswith(b"+zip") or mime_type == b"application/zip":
+        return True
+
+    return False
+
+
+def is_archive_mime_type(mime_type: bytes) -> bool:
+    if mime_type in ARCHIVE_TYPES:
+        return True
+
+    return False
```

Comment:
When you have `if …: return True; return False`, you can often just return the `if` expression instead, assuming it’s a boolean:

```suggestion
    return mime_type in ARCHIVE_TYPES
```

Reply:
Thanks for the suggestion,

I need to work a lot on my coding style 😞 😞